test/files/jvm/duration-tck.scala

```diff
@@ -174,6 +174,20 @@ object Test extends App {
   (1.second + 1.millisecond).unit mustBe MILLISECONDS
 
 
+  // test unit labels
+  for ((unit, labels) ← Seq(
```

Comment:
I think we generally avoid the unicode arrows, so use `<-` instead

Reply:
I found it already at: https://github.com/scala/scala/blob/57698973decdbbe9f058154d1ff586c63c7bb14d/test/files/jvm/duration-tck.scala#L133

I just replaced them all then.